### ISSUE TITLE
Konzept Highscore

### ISSUE THREAD

**Issue by AMatutat** [Programmiermethoden/Dungeon#216]
Wir müssen folgende Fragen beantworten:

- Welche Punkte? Dungeon? Spiel? D2A? 
- Wo anzeigen? Illias? 
- Datenschutz? 
- Wann werden die Werte geupdated? Live? Wöchentlich?

**Comment by AMatutat** [comment]
@AKirsch1 @cagix @malt-r @bcg7 das Ding ist jetzt irgendwie liegen geblieben.


Vorschlag:
Auf der StartPlay wurde uns erzählt, dass Highscores kritischer zu betrachten sind, da der Mehrwert oft nur für die Leute gegeben ist, die weit oben auf der Liste stehen. 
Vielleicht kann man da noch mal tiefer recherchieren und den didaktischen Mehrwert hinterfragen. 
Dann könnten wir im Bericht darauf eingehen und ggf. begründen warum wir das nicht gemacht haben.

**Comment by AKirsch1** [comment]
Man könnte auch noch auf Datenschutzprobleme eingehen.

**Comment by cagix** [comment]
wir sollten irgendein konzept vorweisen können, wie wir technisch zu einem highscore kommen (könnten). dazu die diskussion auf der startplay, angereichert mit problemen aus datenschutzsicht und entsprechende lösungsvorschläge oder eben auch show-stopper.

**Comment by AMatutat** [comment]
> wir sollten irgendein konzept vorweisen können, wie wir technisch zu einem highscore kommen (könnten). dazu die diskussion auf der startplay, angereichert mit problemen aus datenschutzsicht und entsprechende lösungsvorschläge oder eben auch show-stopper.

Das technische Konzept für einen Highscore ist ja recht einfach:

Spiel:
Wir loggen die erreichten Punkte sowieso, also könnte man diese auch a) manuell in das Ilias übertragen (Betrugsrisiko) oder b) per Webkommunikation an einen Server übertragen und dann im Ilias eingeben (Nachteil: Internetanbindung notwendig, vermutlich steht unser Server im VPN, irgendeine Form von Authentifizierung ist notwendig (Datenschutzproblem)).

D2G:
Im Grunde ähnlich. Die Action könnte auch mit einem Webserver kommunizieren. Die Probleme wären ähnlich.
Manuelles Übertragen geht natürlich auch (vermutlich sogar einfacher, da du dir hier als Prof/Tutor die Actionergebnisse angucken kannst).


Show-Stopper wären für mich die Datenschutzbedenken und die notwendige Internetverbindung (das geht irgendwie gegene den "Wir spielen das im Zug" ansatz).

**Comment by cagix** [comment]
ja, genau. das sollte aber so konkret klingen, als wenn wir das so hätten umsetzen wollen ... analog zum datenschutz: das sollte deutlich konkreter klingen und dann einen punkt haben, wo wir aufgehört haben mit weiteren lösungsansätzen.

**Comment by AMatutat** [comment]
> ja, genau. das sollte aber so konkret klingen, als wenn wir das so hätten umsetzen wollen ... analog zum datenschutz: das sollte deutlich konkreter klingen und dann einen punkt haben, wo wir aufgehört haben mit weiteren lösungsansätzen.

hiihi, schau dir mal das Zeichenlimit des Sachbereichts an. Konkret passt da nicht rein.... 
Aber ja. Wir hatten ja auch schon ein paar mal darüber gesprochen, ich find da was .
